src/app/utils.js

```diff
@@ -154,3 +154,12 @@ export const normalizeErrors = (error, code) => {
   }
   return errs;
 };
+
```

Comment:
Debating if this is the right place for it

Reply:
yeah this works!